src/process/process.rs

```diff
@@ -15,15 +15,15 @@ extern {
 }
 
 /// Size of each processes's memory region in bytes
-pub const PROC_MEMORY_SIZE : usize = 8192;
-pub const NUM_PROCS : usize = 2;
+pub const PROC_MEMORY_SIZE : usize = 2048;
+pub const NUM_PROCS : usize = 1;
```

Comment:
Yeah, I think this was a hack to get everything to fit on the nrf. I think 2 * 2048 should work for both platforms until we have a more general fix.